lightcycle-integration-test/build.gradle

```diff
@@ -27,9 +27,12 @@ dependencies {
 
     implementation project(":lightcycle-lib")
 
-    implementation deps.appcompat_v7
+    implementation deps.appcompat_x
+    debugImplementation deps.fragment_testing
```

Comment:
why `debugImplementation deps.fragment_testing`?

Reply:
this is required for `FragmentLoggerTest` to mock fragments related stuff, maybe can make just `implementation` , depends if we want to run test on release flavor